sys/linux/dev_char_raw.txt

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# drivers/char/raw.c
+include <linux/raw.h>
+
+resource fd_char_raw[fd]
+resource fd_char_raw_ctl[fd]
+
+openat$char_raw_ctl(fd const[AT_FDCWD], file ptr[in, string["/dev/raw/rawctl"]], flags flags[open_flags], mode const[0]) fd_char_raw_ctl
+# /dev/raw/raw# only works if ioctl$RAW_CHAR_CTRL_SETBIND executed first.
+syz_open_dev$char_raw(file ptr[in, string["/dev/raw/raw#"]], dev intptr[1:2], flags flags[raw_open_flags]) fd_char_raw
+raw_open_flags = O_RDWR, O_RDONLY, O_APPEND, O_NOATIME, O_NOFOLLOW, O_NONBLOCK, O_PATH, O_EXCL, O_DIRECT
```

Comment:
Are other open_flags from sys.txt applicable to raw devices? Maybe we could just reuse them here?